<launch>
	<param name="/debug" value="True" />

	<node pkg="robot" type="nav_manager.py" name="nav_manager">
	</node>

</launch>